modules/nw-sriov-nic-mlx-secure-boot.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-device.adoc
+
+// :_mod-docs-content-type: PROCEDURE
+[id="nw-sriov-nic-mlx-secure-boot_{context}"]
+= Mellanox Secure Boot
+
+Mellanox (MLX) secure boot supports the option to disable vendor plugins in the SR-IOV Network Operator. This allows {product-title} customers to use the existing virtual functions in the Mellanox cards firmware, and is currently the only way to use Mellanox NICs when secure boot is enabled. Secure boot is required by regulation and desired by customers.
```

Comment:
I think here it should be something like.

The SR-IOV operator supports an option to skip the firmware configuration for Mellanox devices. That is currently the only way to be able and create virtual functions via the SR-IOV Network operator if the system has secure boot enabled.

This will require a manual configuration from the user to allocate the number of virtual functions in the firmware before switching the system to secure boot.


Note: the number of virtual functions in the firmware will be the max number of virtual functions the user can request in the policy (meaning the user can create less)